examples/plot_firing_pattern.py

```diff
@@ -90,8 +107,8 @@
 plt.show()
 
 ###############################################################################
-# Let's plot the soma voltage along with the spiking activity with raster
-# plots and histograms for the Pyramidal cells.
+# Finally, we can plot the soma voltage along with the spiking activity with
+# raster plots and histograms for the Pyramidal cells.
```

Comment:
Nitpick: is "Pyramidal cells" here supposed to link to the source code? If not, it probably doesn't need an uppercase "P".